test/smoke/src/areas/positron/welcome/welcome.test.ts

```diff
@@ -0,0 +1,176 @@
+
+				const expectedInterpreterVersion = new RegExp(`Python ${process.env.POSITRON_PY_VER_SEL}`, 'i');
+				await expect(app.workbench.positronNotebooks.kernelLabel).toHaveText(expectedInterpreterVersion);
+			});
+
+			it('Click on Python console from the Welcome page', async function () {
+				const app = this.app as Application;
+
+				await app.workbench.positronWelcome.newConsoleButton.click();
+				await app.workbench.positronPopups.popupCurrentlyOpen();
+
+				const expectedInterpreterVersion = new RegExp(`Python ${process.env.POSITRON_PY_VER_SEL}`, 'i');
+				await app.workbench.positronPopups.clickOnModalDialogPopupOption(expectedInterpreterVersion);
+
+				// editor is hidden because bottom panel is maximized
+				await expect(app.workbench.editors.editorPart).not.toBeVisible();
+
+				// console is the active view in the bottom panel
+				await expect(app.workbench.positronLayouts.panelViewsTab).toHaveCount(6);
+				await expect(app.workbench.positronLayouts.panelViewsTab.and(app.code.driver.getLocator('.checked'))).toHaveText('Console');
+			});
+		});
+
+		describe('R', () => {
+			before(async function () {
+				await PositronRFixtures.SetupFixtures(this.app as Application);
+			});
+
+			it('Create a new R file from the Welcome page', async function () {
+				const app = this.app as Application;
+
+				await app.workbench.positronWelcome.newFileButton.click();
+
+				await app.workbench.quickinput.selectQuickInputElementContaining('R File');
+
+				await expect(app.workbench.editors.activeEditor.locator(app.workbench.editors.editorIcon)).toHaveClass(/r-lang-file-icon/);
+			});
+
+			it('Click on R console from the Welcome page', async function () {
+				const app = this.app as Application;
+
+				await app.workbench.positronWelcome.newConsoleButton.click();
+				await app.workbench.positronPopups.popupCurrentlyOpen();
+
+				const expectedInterpreterVersion = new RegExp(`R ${process.env.POSITRON_R_VER_SEL}`, 'i');
+				await app.workbench.positronPopups.clickOnModalDialogPopupOption(expectedInterpreterVersion);
+
+				// editor is hidden because bottom panel is maximized
+				await expect(app.workbench.editors.editorPart).not.toBeVisible();
+
+				// console is the active view in the bottom panel
+				await expect(app.workbench.positronLayouts.panelViewsTab.and(app.code.driver.getLocator('.checked'))).toHaveText('Console');
+			});
+
+			it('Create a new R notebook from the Welcome page', async function () {
+				const app = this.app as Application;
+
+				await app.workbench.positronWelcome.newNotebookButton.click();
+
+				await app.workbench.positronPopups.clickOnModalDialogPopupOption('R Notebook');
+
+				await expect(app.workbench.editors.activeEditor.locator(app.workbench.editors.editorIcon)).toHaveClass(/ipynb-ext-file-icon/);
+
+				const expectedInterpreterVersion = new RegExp(`R ${process.env.POSITRON_R_VER_SEL}`, 'i');
```

Comment:
Same comment as above: beware that "R" might already be a part of POSITRON_R_VER_SEL.  If you need "R" and "Python" be part of the regex, maybe we could add validators for the env vars at like here:
https://github.com/posit-dev/positron/blob/main/test/automation/src/positron/fixtures/positronPythonFixtures.ts#L25
and here:
https://github.com/posit-dev/positron/blob/main/test/automation/src/positron/fixtures/positronRFixtures.ts#L25